policy/modules/services/acpi.te

```diff
@@ -135,6 +136,8 @@ domain_dontaudit_list_all_domains_state(acpid_t)
 auth_use_nsswitch(acpid_t)
 
 init_domtrans_script(acpid_t)
+init_telinit(acpid_t)
+init_write_initctl(acpid_t)
```

Comment:
`init_write_initctl()` shouldn't be necessary if you have `init_telinit()`.

Reply:
Removed.